Apps/WebClient/src/ClientApp/src/views/ReleaseNotesView.vue

```diff
@@ -31,6 +31,31 @@ export default class ReleaseNotesView extends Vue {
     <div>
         <BreadcrumbComponent :items="breadcrumbItems" />
         <page-title title="Health Gateway Release Notes" />
+        <release-note
+            date="March 7, 2023"
+            version="v2023.3.07"
+            title="Hospital Visits"
+        >
+            <ul>
+                <li>
+                    We moved the FAQ page to another site. You will find the new
+                    support guide linked in the footer of the website.
+                </li>
+            </ul> </release-note
+        ><release-note
```

Comment:
Nit: There should be a newline between the `><` on line 45, which will result in cleaner auto-formatting.